packages/gatsby-source-wordpress/package.json

```diff
@@ -1,53 +1,102 @@
 {
   "name": "gatsby-source-wordpress",
-  "description": "Gatsby source plugin for building websites using the WordPress CMS as a data source.",
-  "version": "3.12.0-next.0",
-  "author": "Sebastien Fichot <fichot.sebastien@gmail.com>",
+  "description": "Source data from WordPress in an efficient and scalable way.",
+  "author": "Tyler Barnes <tylerdbarnes@gmail.com>",
+  "version": "4.0.0",
   "bugs": {
     "url": "https://github.com/gatsbyjs/gatsby/issues"
   },
-  "bundledDependencies": [],
   "dependencies": {
-    "@babel/runtime": "^7.12.5",
+    "@babel/runtime": "^7.6.3",
+    "@rematch/core": "^1.3.0",
+    "@rematch/immer": "^1.2.0",
+    "@types/cache-manager": "^2.10.3",
+    "@types/ink": "^2.0.3",
+    "async-retry": "^1.3.1",
+    "atob": "^2.1.2",
     "axios": "^0.21.1",
+    "axios-rate-limit": "^1.3.0",
     "better-queue": "^3.8.10",
-    "bluebird": "^3.7.2",
-    "deep-map": "^1.5.0",
-    "deep-map-keys": "^1.2.0",
-    "gatsby-source-filesystem": "^2.12.0-next.0",
-    "json-stringify-safe": "^5.0.1",
-    "lodash": "^4.17.20",
-    "minimatch": "^3.0.4",
-    "qs": "^6.9.4"
+    "btoa": "^1.2.1",
+    "cache-manager": "^3.4.0",
+    "cache-manager-fs-hash": "^0.0.9",
+    "chalk": "^4.1.0",
+    "cheerio": "^1.0.0-rc.3",
+    "clipboardy": "^2.1.0",
+    "diff": "^5.0.0",
+    "dumper.js": "^1.3.1",
+    "execall": "^2.0.0",
+    "fast-json-stable-stringify": "^2.1.0",
+    "file-type": "^15.0.1",
+    "filesize": "^6.1.0",
+    "fs-extra": "^9.0.1",
+    "gatsby-core-utils": "1.11.0-next.0",
+    "gatsby-image": "2.12.0-next.0",
+    "gatsby-plugin-catch-links": "2.11.0-next.0",
+    "gatsby-source-filesystem": "2.12.0-next.0",
+    "glob": "^7.1.6",
+    "got": "^11.7.0",
+    "graphql-query-compress": "^1.2.2",
+    "lodash": "^4.17.19",
+    "node-fetch": "^2.6.0",
+    "p-queue": "^6.3.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
move to a peerdependency. Gatsby currently supports `16.4.0` can we support it? Or does the plugin uses hooks anywhere? If react is only used in tests we should move it to devDeps

Reply:
React isn't used anywhere - I can't remember why this is in here :p I'll remove it. If Gatsby has a peerdep on React already I don't think there's a reason for this plugin to have it listed. This plugin doesn't actually do anything related to React now that I'm thinking about it 🤔